contracts/libraries/Portfolio.sol

```diff
@@ -178,6 +198,34 @@ library Portfolio {
         }
     }
 
+    /// @dev Decrease the most mature position's bondAmount by the amount
+    ///      specified. If the amount is equal to the position's remaining
+    ///      bondAmount, remove the position.
+    /// @param _amount Amount of bonds to remove from the position.
+    /// @return value A copy of the position that was just popped.
+    function _decreasePosition(
+        State storage self,
+        uint128 _amount
+    ) internal returns (IEverlong.Position memory value) {
+        unchecked {
```

Comment:
What is the unchecked doing here? As far as I can tell, none of the internal operations would use any checked math.

Reply:
Derp, you're totally right. Leftover copypasta